src/QA/QA.Web/Models/WithdrawVacancy/PostFindVacancyEditModelResult.cs

```diff
@@ -4,7 +4,8 @@ public enum PostFindVacancyEditModelResultType
     {
         NotFound,
         AlreadyClosed,
-        CanClose
+        CanClose,
+        CannotClose
```

Comment:
Do we need to differentiate between cannot close and already closed? Perhaps the messages are different in which case, can you rename cannot close to NotLive?